pkg/cli/check_update.go

```diff
@@ -14,7 +14,7 @@ func CheckUpdate() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:               "update",
+		Use:               "update [flags] [config[.yaml]...]",
```

Comment:
```suggestion
		Use:               "update [config[.yaml]...]",
```

nit: cobra adds the `[flags]` bit automatically, we can just add the config file part.